### PR TITLE
Amp 794 ina updates

### PR DESCRIPTION
**Pull request by dfische3**
Updates INA speech segmenter to roll 10 seconds or less of noise/silence in with the previous segment.  Also updates the keep segments to remove noise as well as ignore blocks of silence/noise of greater than  60 seconds.  Lastly,  HPC tools were updated to convert output to AMP standardized json.

### CODE REVIEW

File: tools/amp_segment/keep_speech.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
   <help>
 .. class:: infomark
 
-Remove segments of silence and music from an audio file.
+Remove segments of silence, noise and music from an audio file.
 
   </help>
 </tool>
```
Comment (yingfeng-iu) [inline tools/amp_segment/keep_speech.xml:30]: Let's keep the original remove_silence_music MGM intact (including its python scipts), as they are being used in staging in some unfinished workflows. We can add new tool keep_speech along with its own python code. The old tools will be removed from tool panel, and we can inform create new workflows with new tool.

File: tools/amp_stt/kaldi_transcript_to_amp_transcript.py [hunk 2]
```diff
@@ -11,8 +11,7 @@
 from speech_to_text import SpeechToText, SpeechToTextMedia, SpeechToTextResult, SpeechToTextScore, SpeechToTextWord
 
 # Convert kaldi output to standardized json
-def main():
-	(media_file, kaldi_file, kaldi_transcript_file, output_json_file) = sys.argv[1:5]
+def convert(media_file, kaldi_file, kaldi_transcript_file, output_json_file):
 	mgm_utils.exception_if_file_not_exist(kaldi_file)
 	mgm_utils.exception_if_file_not_exist(kaldi_transcript_file)
 	results = SpeechToTextResult()
```
Comment (yingfeng-iu) [inline tools/amp_stt/kaldi_transcript_to_amp_transcript.py:17]: There is a change merged in master on line 17. try synch and see if any conflict.

**Review by yingfeng-iu — APPROVED** [review]
looks good! I made a suggestion on keeping old MGM. also, would be helpful to add some logging at end of each important stage to help debugging. I added some along the way when going through MGMs on empty inputs. let's add more as we see fit. Also, i found that in try-except block, adding exception to the print statement doesn't print stacktrace (unlike log in Java), so i added traceback.print_exc() in each error case. Let's keep that pattern as well. Without stacktrace it's hard to locate the error.

File: tools/amp_segment/keep_speech.xml (hunk 1 above)
Comment (dfische3) [reply tools/amp_segment/keep_speech.xml:30]: Good thought.  I added the old ones back in, but kept the reference in tool_conf.xml to the new one.

**Pull request merged**